nginx/repo.sls

```diff
@@ -7,9 +7,9 @@ nginx_repo:
   pkgrepo.managed:
     - humanname: Nginx PPA
     {%- if mainline == True %}
-    - name: deb http://nginx.org/packages/mainline/{{ grains['os']|lower }}/ {{ lsb_codename }} nginx
+    - name: deb [arch=amd64] http://nginx.org/packages/mainline/{{ grains['os']|lower }}/ {{ lsb_codename }} nginx
     {%- else %}
-    - name: deb http://nginx.org/packages/{{ grains['os']|lower }}/ {{ lsb_codename }} nginx
+    - name: deb [arch=amd64] http://nginx.org/packages/{{ grains['os']|lower }}/ {{ lsb_codename }} nginx
```

Comment:
While you're at it you may want to change this to `https` as well.

Reply:
done